packages/darklang/languageTools/semanticTokens.dark

```diff
@@ -276,6 +276,7 @@ module Darklang =
             tokenizeDefinition t.definition ]
           |> Stdlib.List.flatten
 
+      // TODO: maybe this should be inside `Expr` module?
```

Comment:
```suggestion
```

I think it's good as-is, unless we refactor other parts of our codebase similarly - more accessible this way